documentation/modules/ROOT/pages/assemblies/cdc-mysql-connector/as_connector-common-issues.adoc

```diff
@@ -11,10 +11,10 @@
 
 toc::[]
 
-include::../../modules/cdc-mysql-connector/r_configuration-and-startup-errors.adoc[leveloffset=+1]
+include::{partialsdir}/modules/cdc-mysql-connector/r_configuration-and-startup-errors.adoc[leveloffset=+1]
```

Comment:
Where is `partialsdir` defined? Does it exist implicitly?

Reply:
It is one of the attributes Antora provides implicitly. An Antora aware editor would provide it as well.

It is also set in _attributes.adoc for non-Antora environments, supposedly downstream. As include supports absolute file names, it might be an option for a non-Antora (downstream) environment to pass this at build time as an absolute path. 
Using relative paths is probably messy. / cc: @bhardesty

Reply:
Ok, gotcha. Thanks a lot for clarifying!